Sources/PostgresNIO/Data/PostgresDataType.swift

```diff
@@ -227,3 +222,13 @@ public struct PostgresDataType: Codable, Equatable, ExpressibleByIntegerLiteral,
         return self.knownSQLName ?? "UNKNOWN \(self.rawValue)"
     }
 }
+
+// TODO: The Codable conformance does not make any sense. Let's remove this with next major break.
+extension PostgresDataType: Codable {}
+
+// TODO: The ExpressibleByIntegerLiteral conformance does not make any sense and is not used anywhere. Remove with next major break.
```

Comment:
My guess would be that Tanner probably used this during dev to be able to just drop raw data type codes in for quick testing 'cause it felt more convenient than having to add the `.init()` boilerplate (I can understand the feeling! 😅). Of course, then he either just forgot about it later or looked at it and went "well why not just leave it in" - he was happy to let me take the role of overzealous code neat freak 😆

(I'm 100% in favor of ditching it regardless of all that, all it does is make it easier to misuse the API.)